gigasecond/gigasecond_test.spec.js

```diff
@@ -0,0 +1,23 @@
+import Gigasecond from './gigasecond';
+
+describe('Gigasecond', () => {
+
+  it('tells the gigasecond birth of a living person', () => {
+    let gs = new Gigasecond(new Date(1977, 5, 13));
+    let expectedDate = new Date(2009, 1, 19);
+    expect(gs.date()).toEqual(expectedDate);
+  });
+
+  xit('tells gigasecond birth of a future person', () => {
```

Comment:
If the only thing that matters is the day, in the ruby example, these both tests should assert the same returned value:

1. `test_full_time_specified`
2. `test_full_time_with_day_roll_over`

Because both constructor parameter `Time.utc(2015, 1, 24, 22, 0, 0)` and `Time.utc(2015, 1, 24, 23, 59, 59)` should be converted to a value equal to `Time.utc(2015, 1, 24, 0, 0, 0)` (up to the day matters) and `Gigasecond.from()` should return the same value.